sdk/servicebus/azure-servicebus/samples/async_samples/batch_delete_session_async.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Example to show sending message(s) to a Service Bus Queue.
+"""
+
+import os
+import asyncio
+from datetime import datetime, timezone, timedelta
+from azure.servicebus.aio import ServiceBusClient
+from azure.servicebus import ServiceBusMessage, NEXT_AVAILABLE_SESSION
+
+
+CONNECTION_STR = os.environ['SERVICEBUS_CONNECTION_STR']
+SESSION_QUEUE_NAME = os.environ["SERVICEBUS_SESSION_QUEUE_NAME"]
+
+async def send_single_message(sender, i):
+    message = ServiceBusMessage(f"Single Message {i}", session_id="0")
+    await sender.send_messages(message)
+
+async def run():
+    servicebus_client = ServiceBusClient.from_connection_string(conn_str=CONNECTION_STR)
+    async with servicebus_client:
+        sender = servicebus_client.get_queue_sender(queue_name=SESSION_QUEUE_NAME, session_id=NEXT_AVAILABLE_SESSION)
+        async with sender:
+            await send_single_message(sender, 1)
+            await send_single_message(sender, 2)
+            await send_single_message(sender, 3)
+        print(f"All messages sent before {datetime.now(timezone.utc)}")
+
+
+        receiver = servicebus_client.get_queue_receiver(queue_name=SESSION_QUEUE_NAME, session_id=NEXT_AVAILABLE_SESSION)
```

Comment:
is there a difference in usage/behavior b/w sessionful q delete and non-sessionful q?